src/Microsoft.AspNetCore.Server.Kestrel.Core/Microsoft.AspNetCore.Server.Kestrel.Core.csproj

```diff
@@ -10,6 +10,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>CS1591;$(NoWarn)</NoWarn>
     <EnableApiCheck>false</EnableApiCheck>
+    <!-- TODO: remove when https://github.com/Microsoft/msbuild/issues/1293 is fixed -->
+    <GenerateResourceMSBuildRuntime Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">CurrentRuntime</GenerateResourceMSBuildRuntime>
```

Comment:
No need for the condition, since these projects don't target (desktop) .NET 2.0/3.5.